src/control/system/raft/database_members.go

```diff
@@ -22,8 +23,8 @@ type (
 	MemberRankMap map[ranklist.Rank]*system.Member
 	// MemberUuidMap provides a map of UUID->*system.Member.
 	MemberUuidMap map[uuid.UUID]*system.Member
-	// MemberAddrMap provides a map of string->[]*system.Member.
-	MemberAddrMap map[string][]*system.Member
+	// MemberAddrMap provides a map of string->system.Members.
+	MemberAddrMap map[string]system.Members
```

Comment:
Has this change been verified to work ok with snapshots written from before the change? `system.Members` is a type alias, so it will _probably_ work OK, but it would be a really good idea to generate a fixture with the older format and make sure that it unmarshals correctly on snapshot load. Otherwise you're going to be introducing an incompatibility (e.g. 2.6.4 server can't read 2.6.3 snapshots).